packages/core/src/server/utils/custom-chunk-extractor.ts

```diff
@@ -0,0 +1,26 @@
+import { ChunkExtractor } from "@loadable/server";
+
+/**
+ * Custom ChunkExtractor class to override `preload` with `modulepreload` based
+ * on asset filename.
+ */
+export class CustomChunkExtractor extends ChunkExtractor {
+  /**
+   * The `getPreAssets` method.
+   *
+   * @returns A list of assets.
+   */
+  getPreAssets() {
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    const assets = (ChunkExtractor as Function).prototype.getPreAssets.call(
+      this
+    );
```

Comment:
Hey, couldn't `super` be used here to call that method in the parent prototype chain? I think it's possible to write something like this:

```suggestion
    const assets = super.getPreAssets();
```

Reply:
Oh indeed that's possible. Need to check it locally first though. Have you tried it? I'm worried that the `ChunkExtractor` ends up being exported from the package as a function not as a `class` so maybe the `super.` syntax would not work in that case?

Reply:
Yeah, no need to worry about it it seems it's a `class` [look here](https://unpkg.com/browse/@loadable/server@5.14.2/lib/ChunkExtractor.js)

Reply:
I didn't try it but I had a gut feeling (from my old OOP background, I guess :smile:). Glad it worked. :raised_hands:

Reply:
Alrighty! Fixed it @DAreRodz! Thank you!